definitions/20240910_definition_cookietona.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Cookietona"
```

Comment:
This is fun, but doesn't make sense. lol :D

Please add some real definitions if they are needed to support your article.